codegen/data.go

```diff
@@ -147,6 +170,30 @@ func BuildData(cfg *config.Config) (*Data, error) {
 		// otherwise show a generic error message
 		return nil, fmt.Errorf("invalid types were encountered while traversing the go source code, this probably means the invalid code generated isnt correct. add try adding -v to debug")
 	}
+	aSources := []AugmentedSource{}
+	for _, s := range cfg.Sources {
+		wd, err := os.Getwd()
```

Comment:
I'm not happy with having this here, but it seemed like the most reliable way to build the full path to the sources. Is there a better way?